node/core/approval-voting/src/lib.rs

```diff
@@ -1366,8 +1375,19 @@ async fn handle_approved_ancestor(
 		},
 	);
 
+	// `reverse()` to obtain the ascending order from highest to lowest
```

Comment:
```suggestion
	// `reverse()` to obtain the ascending order from lowest to highest
```

I assume this was meant?